src/ScheduleService.Business/Group/FindGroupsCommand.cs

```diff
@@ -0,0 +1,15 @@
+﻿using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.ScheduleService.Business.Group.Interfaces;
+using LT.DigitalOffice.ScheduleService.Models.Dto.Requests.Group;
+using LT.DigitalOffice.ScheduleService.Models.Dto.Responses.Group;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.ScheduleService.Business.Group;
+
+public class FindGroupsCommand : IFindGroupsCommand
```

Comment:
rename to GetGroupsCommand(Don't use find despite the fact it was used in DepartmentService)

Reply:
+